client/cli/src/config.rs

```diff
@@ -538,6 +535,12 @@ pub trait CliConfiguration<DCV: DefaultConfigurationValues = ()>: Sized {
 		Ok(self.shared_params().log_filters().join(","))
 	}
 
+
+	/// Is log reloading disabled
```

Comment:
You should comment what's the default to stay consistent with the other methods.